web/client-api/src/main/java/io/deephaven/web/public/treegrid.js

```diff
@@ -34,7 +45,7 @@ class TreeGrid {
       if (cellElt.classList.contains('expanded')) {
         this.treeTable.collapse(rowNumber);
       } else {
-        this.treeTable.expand(rowNumber);
+        this.treeTable.expand(rowNumber, false);
```

Comment:
Param is optional, don't need to provide `false` here.
```suggestion
        this.treeTable.expand(rowNumber);
```